.storybook/stories/useProgress.stories.tsx

```diff
@@ -46,11 +46,27 @@ function LoadExtras() {
 
 function UseProgressScene() {
   return (
-    <React.Suspense fallback={<Loader />}>{boolean('Load extras', false) ? <LoadExtras /> : <Helmet />}</React.Suspense>
+    <React.Suspense fallback={<CustomLoader />}>
+      {boolean('Load extras', false) ? <LoadExtras /> : <Helmet />}
+    </React.Suspense>
   )
 }
 
 export const UseProgressSceneSt = () => <UseProgressScene />
 UseProgressSceneSt.story = {
   name: 'Default',
 }
+
+export function WithOutOfTheBoxLoader() {
+  return (
+    <React.Suspense
+      fallback={
+        <Html>
+          <Loader />
+        </Html>
+      }
+    >
+      {boolean('Load extras', false) ? <LoadExtras /> : <Helmet />}
+    </React.Suspense>
+  )
+}
```

Comment:
Tiny nitpick, this could be the default and the other one could be "With custom Loader"